pandas/tests/groupby/test_quantile.py

```diff
@@ -343,3 +343,54 @@ def test_columns_groupby_quantile():
     )
 
     tm.assert_frame_equal(result, expected)
+
+
+def test_timestamp_groupby_quantile():
+    # GH 33168
+    df = DataFrame(
+        {
+            "timestamp": pd.date_range(
+                start="2020-04-19 00:00:00", freq="1T", periods=200, tz="UTC"
+            ).floor("1H"),
+            "category": list(range(1, 201)),
+            "value": list(range(201, 401)),
+        }
+    )
+
+    result = df.groupby("timestamp").quantile([0.1, 0.5, 0.9])
+
+    expected = DataFrame(
+        [
+            {"category": 6.9, "value": 206.9},
```

Comment:
Is it possible to create the DataFrames with less data? If possible, the test should be minimal

Reply:
sure thing @phofl, have updated the testcase accordingly. do let me know if any other change is needed. Thanks in advance.